recipes/detecto/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "detecto" %}
+{% set version = "1.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/detecto-{{ version }}.tar.gz
+  sha256: 255f29b8970f14d686f6ea3751d26a3c3d74724a8ed3371eb7ca27967e2e67ab
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib-base
+    - opencv
```

Comment:
Upstream depends on https://github.com/opencv/opencv-python 
Not sure which of the conda-forge opencv versions is the best choice here?